core/src/main/java/com/linecorp/armeria/server/DefaultServerConfig.java

```diff
@@ -823,6 +831,8 @@ static String toString(
         buf.append(absoluteUriTransformer);
         buf.append(", unhandledExceptionsReportIntervalMillis: ");
         buf.append(unhandledExceptionsReportIntervalMillis);
+        buf.append(", serverMetrics: ");
+        buf.append(serverMetrics);
```

Comment:
We should also implement `toString()` for `serverMetrics` if we want to do this.

Reply:
oh i will implement toString() method!! thank you!!